ci/citest/acceptance-test/multibox/config.source

```diff
@@ -16,6 +16,11 @@ PREFIX="24"
 BOXES=(
     "minimal-7.2.1511-x86_64.kvm.box"
 )
+ZK_NODES=(
+  "10.0.100.10"
+  "10.0.100.11"
+  "10.0.100.12"
```

Comment:
Sorry, I misread the code. I saw the echo but missed the nc command. My bad.

Could I read up anywhere about what this sending "srvr" to tcp port 2181 does exactly?